package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@toyota-research-institute/lakefront",
   "description": "React UI Components",
-  "version": "0.73.0",
+  "version": "0.69.4",
```

Comment:
the version seems to be going back, maybe you need to first merge main?

Reply:
Yeah, this looks like you were working on an earlier version of the codebase before merging to main. In this case, you have to manually update the version (automatic bumping doesn't handle retro version updates well).